### PR TITLE
SPCTR-2965: Fix - Call-to-action block heading has extra margin in backend.

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Unwanted margin on the top of the H3 tag in the editor of page with Spectra One theme.

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +166,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Call-to-action - Resolved extra margin-top issue in heading of call-to-action block in spectra-one theme.
```
Comment (bsf-zanev) [inline README.md:170]:
```suggestion
* Fix: Call To Action - Resolved an issue wherein an extra margin would appear above the heading of the Call To Action block in the Spectra One theme.
```

File: readme.txt [hunk 2]
```diff
@@ -166,6 +166,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 == Changelog ==
 
+= x.x.x =
+* Fix: Call-to-action - Resolved extra margin-top issue in heading of call-to-action block in spectra-one theme.
```
[the remark this answers is not in the document]
Comment (bsf-zanev) [reply readme.txt:170]: Agreed regarding the caps - I've added a refined changelog entry above. Resolving this comment.

CC - @aapatre

**Review by bsf-zanev — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]
@rajmohanreddy774 Please resolve merge conflicts and check if above mentioned comments are resolved.

**Review by AbhijitNage123 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: updated.

**Pull request merged**